src/main/java/it/pagopa/afm/marketplacebe/util/CommonUtil.java

```diff
@@ -29,6 +29,6 @@ public <T> PageInfo buildPageInfo(Page<T> page) {
      */
     public boolean isValidityDateToAcceptable(LocalDate validityDateTo) {
         LocalDate now = LocalDate.now();
-        return validityDateTo == null || !(validityDateTo.isEqual(now) && validityDateTo.isBefore(now));
+        return validityDateTo == null || !(validityDateTo.isEqual(now) || validityDateTo.isBefore(now));
```

Comment:
this condition is always `true` except when `validityDateTo` equals to `now`

Reply:
se `validityDateTo = now`: `F || !(T || F) => F`
se `validityDateTo = before`: `F || !(F || T) => F`
se `validityDateTo = after`: `F || !(F || F) => T`
se `validityDateTo = null`: `T || !(F || F) => T`

I think it is correct

Reply:
Sorry, i missed the brackets